bin/dump_data.sh

```diff
@@ -61,32 +69,62 @@ for app in "${APPS[@]}"; do
 done
 
 dump_schema() {
-  echo "Dumping schema to $1..."
-  pg_dump --schema-only -f "$1"
+    echo "Dumping schema to $1..."
+    pg_dump --schema-only -f "$1"
 }
 
 dump_data() {
-  echo "Dumping data to $1..."
-  pg_dump "${INCLUDES[@]}" --disable-triggers --data-only > "$1"
+    echo "Dumping data to $1..."
+    pg_dump "${INCLUDES[@]}" --disable-triggers --data-only >"$1"
 }
 
 append_data() {
-  echo "Appending data to $1..."
-  pg_dump "${INCLUDES[@]}" --disable-triggers --data-only \
-    | sed '/^SET\|^SELECT pg_catalog.set_config/d' >> "$1"
+    echo "Appending data to $1..."
+    pg_dump "${INCLUDES[@]}" --disable-triggers --data-only |
+        sed '/^SET\|^SELECT pg_catalog.set_config/d' >>"$1"
 }
 
+dump_csv() {
+    echo "Dumping data to csv..."
+
+    WHERE_CLAUSE=""
+    for app in "${APPS[@]}"; do
+        if [ -n "$WHERE_CLAUSE" ]; then
+            WHERE_CLAUSE+=" OR "
+        fi
+        WHERE_CLAUSE+="tablename LIKE '${app}_%'"
+    done
+
+    TABLES=$(psql -Atc "SELECT tablename FROM pg_tables WHERE schemaname='public' AND ($WHERE_CLAUSE);")
+
+    for table in $TABLES; do
+        echo "dumping $table..."
+        psql -c "\copy $table TO '$CSV_OUTPUT_DIR/$table.csv' WITH CSV HEADER"
+    done
+
+    zip -j "$ZIP_FILE" "$CSV_OUTPUT_DIR"/*.csv
+    rm -f "$CSV_OUTPUT_DIR"/*.csv
+}
+
+if $CSV; then
+    if [ ! -d "$CSV_OUTPUT_DIR" ]; then
+        echo "csv output directory $CSV_OUTPUT_DIR does not exist in current path"
+    else
+        dump_csv
+    fi
+    exit 0
+fi
```

Comment:
Can't you instead do `mkdir -p $CSV_OUTPUT_DIR` and remove it after the zip is generated? That way we don't have to include the `csv_dumps` dir in the git repo